### PR TITLE
Cv page

### PR DESCRIPTION
**Pull request by YechiamW3**
WIP

### CODE REVIEW

File: cv_next/app/cv/[cvId]/components/pageWrapper.tsx
Comment (avnogy) [inline cv_next/app/cv/[cvId]/components/pageWrapper.tsx:1]: can we delete this?

File: cv_next/app/cv/[cvId]/components/cvPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import generateImageUrl from "@/helpers/imageURLHelper"
+import { ClientCvModel } from "@/types/models/cv";
+import Image from "next/image"
+
+
+export const CvPreview = ({cv}: {cv: ClientCvModel}) => {
+
+    const imageUrl = generateImageUrl(cv.documentLink);
+
+    //TODO: add info somewhere?
```
Comment (avnogy) [inline cv_next/app/cv/[cvId]/components/cvPreview.tsx:10]: what do you mean?

File: cv_next/app/actions/comments/commentLikes.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+"use server";
+
+import { updateComment } from "@/server/api/comments";
+import CommentModel, { ClientCommentModel } from "@/types/models/comment";
+
+export const addLikeToComment = async (comment: ClientCommentModel, userId: string)
```
Comment (avnogy) [inline cv_next/app/actions/comments/commentLikes.ts:6]: maybe we should refactor this file cause these two functions look almost the same

**Comment by Aradon267** [comment]
Closing this as the branch should be #41

**Pull request closed without merging**